adapters/oidc/spring-boot2/pom.xml

```diff
@@ -31,8 +31,8 @@
   <description/>
 
   <properties>
-    <spring-boot.version>2.0.5.RELEASE</spring-boot.version>
-    <spring.version>5.0.2.RELEASE</spring.version>
+    <spring-boot.version>2.4.13</spring-boot.version>
+    <spring.version>5.3.13</spring.version>
```

Comment:
I believe these two lines shouldn't be changed. All dependencies that use these are either `provided` or `optional` therefore the project that uses the adapter decides what version of dependencies are used (which means we test it with the correct version regardless of the version specified here). 

The version specified here just guarantees backward compatibility with the older version which is I believe what we want. For example, if someone uses an older version of SB, we don't want to break anything for them in the future version of the adapter and the version specified here helps us to enforce it.

Reply:
@mhajas Thanks for your review. Here's a MR to the pipeline repo[1].

Regarding the versions for Spring Boot 2 adapter; you're probably right. However, I've thought when we test only 2.4 and 2.6 version, those versions for the adapter itself might be replaced as well. 

[1] https://keycloak-gitlab.com/keycloak/keycloak-pipeline/-/merge_requests/89